packages/web-components/package.json

```diff
@@ -39,6 +38,7 @@
   "devDependencies": {
     "@axe-core/puppeteer": "^4.10.0",
     "@babel/core": "^7.12.13",
+    "@department-of-veterans-affairs/css-library": "^0.12.2",
```

Comment:
This was originally in the `dependencies` list but because this repo is set up using workspaces, I could not update the USWDS package the css-library without also needing it done in the web-components packages. We have a [separate ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3401) for that work.